app/services/reporting/total_user_count_report.rb

```diff
@@ -44,18 +57,40 @@ def verified_user_count
       end
     end
 
+    def new_verified_user_count
+      Reports::BaseReport.transaction_with_timeout do
+        Profile.where(active: true).where(activated_at: current_month).count
```

Comment:
Probably an edge case, but would this double-count a user who verified and then re-verified their identity in the same month?

Reply:
a user can only have one active profile at a time, and we even have that enforced w/ a `[user_id, active]` unique index in the DB, so it shouldn't double count those users.

Reply:
Ah, I think I had a brain fart and overlooked the `active` filter.